kotlin/src/main/kotlin/io/logto/sdk/core/exception/LogtoException.kt

```diff
@@ -0,0 +1,24 @@
+package io.logto.sdk.core.exception
+
+open class LogtoException(
+    message: String,
+    cause: Throwable? = null,
+) : RuntimeException(message, cause) {
+    class VerificationException(
+        message: Enum<Verification>,
+        cause: Throwable? = null,
+    ) : LogtoException(message.name, cause)
+
+    enum class Verification {
+        INVALID_URI_FORMAT,
+        URI_MISMATCHED,
+        MISSING_CODE_URI_PARAMETER,
+        MISSING_STATE_URI_PARAMETER,
+        STATE_MISMATCHED,
+    }
+
+    class RedirectUriReturnedException(
```

Comment:
Do we need an extra class explicitly for this case? Or just map the error and errorDesc values to the general verificationException you have defined above.